student.rb

```diff
@@ -1,11 +1,10 @@
 require_relative 'person'
 
 class Student < Person
-  attr_reader :classroom
-
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(name, age, parent_permission)
+    super(age, name, parent_permission)
```

Comment:
- [ ] as part of the requirements kindly add attr_reader :classroom and make sure to add `a method that serves as the 'belongs-to' side (a student belongs to a classroom)` example : 
```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```